core/templates/dev/head/pages/exploration_editor/simple_editor_tab/questions/text_interaction_directive.html

```diff
@@ -0,0 +1,21 @@
+<editor-plaintext-field original-value="getAnswer()"
+                   on-finish-editing="saveAnswer(newValue)"
```

Comment:
Align left end of each attr name with the one on line 1. Follow existing style.

Reply:
Done.